syncstorage/views/__init__.py

```diff
@@ -213,12 +215,11 @@ def get_info_configuration(request):
     else:
         LIMIT_NAMES = (
             "max_request_bytes",
+            "max_record_payload_bytes",
         )
     limits = {}
     for name in LIMIT_NAMES:
         limits[name] = get_limit_config(request, name)
-    # This limit is hard-coded for now.
-    limits["max_record_payload_bytes"] = MAX_PAYLOAD_SIZE
```

Comment:
I moved this under the control of `get_limit_config` so that it can be adjusted via config file, which is important for the memcached tests.